src/schema/artwork/__tests__/artwork.test.js

```diff
@@ -937,4 +938,96 @@ describe("Artwork type", () => {
       })
     })
   })
+
+  describe("Attribution class", () => {
+    keys(attributionClasses).forEach(attributionClass => {
+      it(`returns proper attribution class name for ${attributionClass} artwork`, () => {
+        artwork.attribution_class = attributionClass
+        const query = `
+          {
+            artwork(id: "richard-prince-untitled-portrait") {
+              attribution_class {
+                name,
+              }
+            }
+          }
+        `
+        return runQuery(query, rootValue).then(data => {
+          expect(data).toEqual({
+            artwork: {
+              attribution_class: {
+                name: attributionClasses[attributionClass].name,
```

Comment:
In general I think it's preferable to have a more explicit assertion in a spec, as inadvertent `nil` or `undefined` values creeping in can frequently cause tests to still pass (in this style).

Also, I love how each class is tested, but that feels a bit much, maybe we can just have one explicit test for a specific attribution type?